Refresh.GameServer/Startup.cs

```diff
@@ -16,7 +18,7 @@
 BunkumHttpServer server = new()
 {
     AssumeAuthenticationRequired = true,
-    UseDigestSystem = true,
+    // UseDigestSystem = true,
```

Comment:
is there a reason this is just commented out and not removed outright?

Reply:
Yeah, just a fragment from the process of updating Bunkum. This option no longer exists so I'll remove it.